examples/bri-3/src/bri/identity/bpiSubjectAccounts/models/bpiSubjectAccount.ts

```diff
@@ -1,14 +1,12 @@
-import { IBpiSubject } from "../../bpiSubjects/models/bpiSubject.interface";
-import { IBpiSubjectAccount } from "./bpiSubjectAccount.interface";
+import { BpiSubject } from '../../bpiSubjects/models/bpiSubject';
 
-export class BpiSubjectAccount implements IBpiSubjectAccount {
+export class BpiSubjectAccount {
     id: string; // TODO: Add uuid after #491
```

Comment:
Please make all these fields explicitly private. We want the state of the domain objects to be changed only through methods.

Reply:
done